iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -61,6 +61,9 @@ void setTranslationInfo(FuncOp entryPointFn,
 
 SmallVector<unsigned> getPartitionedLoops(Operation *op) {
   SmallVector<unsigned> partitionedLoops;
+  if (auto mmt4dOp = dyn_cast<linalg::Mmt4DOp>(op)) {
```

Comment:
Unfortunately you need to do the same at the dispatch region creation method here (https://github.com/google/iree/blob/92414566d7d56df9b5a654ea232c37bea9ab2b44/iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp#L658). I should fine a common place for it (or if you want to find a common place for it, that would be awesome!)

Reply:
I think this can be a Flow level lib (e.g `Flow/DispatchUtils`) where we made distribution decisions. later tiling related properties are  codegen specific then will exist in codegen libs

Reply:
Ok, Ill stamp this, but do you mind moving it while you are at it.

Reply:
IMO It easier to land this then refactor & merge `Flow/Transforms/DispatchLinalgOnTensors.cpp & codegen/utils/utils.cpp` -> `Flow/DispatchUtils`